CHANGELOG.md

```diff
@@ -8,6 +8,14 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## [Unreleased]
 
+### Added
+
+- Add `go.opentelemetry.io/otel/sdk/metric/exemplar` package which includes
+  `Exemplar`, `Filter`, `SampledFilter`, `AlwaysOnFilter`,
+  `HistogramReservoir`, `FixedSizeReservoir`, `Reservoir`, `Value` and
+  `ValueType` types. These will be used for configuring the exemplar reservoir
+  for the metrics sdk. (#5747)
```

Comment:
The style of this document is wrapped on sentence. Can we stick wit this style? Maybe add the types in a bulleted list if they need to be collapsed.

Reply:
done